### PR TITLE
Update application and build operators image references

### PR DESCRIPTION
**Pull request by mmorhun**

### CODE REVIEW

File: components/build/build-service/kustomization.yaml [hunk 1]
```diff
@@ -16,12 +16,9 @@ patchesJSON6902:
       value: build-service
 
 images:
-- name: controller
+- name: quay.io/redhat-appstudio/build-service
```
Comment (Michkov) [inline components/build/build-service/kustomization.yaml:19]: The name is `quay.io/redhat-appstudio/build-operator`

Comment (mmorhun) [reply components/build/build-service/kustomization.yaml:19]: Fixed reference on build-service side and updated commit reference

**Comment by Michkov** [comment]
Included MRs:
- https://github.com/redhat-appstudio/managed-gitops/pull/108
- https://github.com/redhat-appstudio/application-service/pull/107

This enables gitops workflow

**Review by Michkov — APPROVED** [review]

**Pull request merged**